### ISSUE TITLE
feature request: auto download covers

### ISSUE THREAD

**Issue by briantho** [Aurelio92/RVLoader#18]
Is it possible to implement this, so that covers for games can be downloaded directly from the wii console?

**Comment by loopj** [comment]
Most people use RVLoader in portables, and I think 99% of people don't include the WiFi module in their builds so I'm not sure how many people would want this. Are you using RVLoader in a trimmed Wii?

**Comment by briantho** [comment]
No, I'm using on a normal wii. I'm primary using this instead of usb loaders such as usb loader gx and wiiflow, for 2 primary reasons. The reasons are no need to muck around with cios, just homebrew channel, and gc2wiimote support, which is a huge thing for me. I've done usb loaders in the past with my previous wii. IMHO, RVLoader is very nice, just missing some features. I didn't realize most people remove the wifi chip from the wii when making a portable console. Why? does it really save that much space, or something? I'm just wondering? anyway, maybe I'm overthinking this. Do you know if there is some sort of cover art downloader for pc. Worst case is i could just use gametdb, but i don't want to go through every game I have manually and download a cover.

**Comment by loopj** [comment]
Most people don't bother relocating the WiFi module in a portable since Nintendo shut down all their online services for the Wii. Some people definitely do relocate it, but I would say the small minority.

There are a couple of desktop applications that actually help to fetch cover art for Wii and Gamecube games, you can check them out here https://www.gametdb.com/Wii/downloads hopefully that helps

**Comment by fanXbox** [comment]
It would not be bad if the covers, in addition to being loaded from the current folder, are also loaded from the USBloader gx folders.
PS: for me it would be very convenient and I would avoid duplication
